apps/i18n/common/en_us.json

```diff
@@ -961,6 +962,7 @@
   "standardMappings": "Standard Mappings",
   "standardsAndFramework": "Curriculum Standards",
   "standardsAndFrameworkDescription": "Information about curriculum standards alignment for Code.org courses.",
+  "started": "Started",
```

Comment:
[spec](https://docs.google.com/document/d/1MJeQjVbNqprZWqypYcN68eWkZ2G-HHx8PiqPK73N2aw/edit#heading=h.2h76yhj6mt9r) says "started" for the summary view legend, but "in progress" for the detail view legend.  Should be consistent - do we have a preference?

Reply:
I'm guessing we want "in progress" but that's a @poorvasingal question.